Problem039/C++/Makefile

```diff
@@ -1,7 +1,7 @@
-CXX = g++
```

Comment:
I [renamed](https://github.com/DestructHub/ProjectEuler/pull/69/commits/7f3da85f1a38734acf151ab4687657e8d461fd27) the directory. It was an someone's error.